app/models/manager_refresh/inventory_collection_default/cloud_manager.rb

```diff
@@ -0,0 +1,226 @@
+class ManagerRefresh::InventoryCollectionDefault::CloudManager < ManagerRefresh::InventoryCollectionDefault
```

Comment:
I think this should be at `app/models/manageiq/providers/inventory/targets/cloud_manager` - then it's closer to the actual models